### ISSUE TITLE
Disbanding Nation from lowered Resident population can cause Capital to display Null value on /town (name)

### ISSUE THREAD

**Issue by KingBohica** [TownyAdvanced/Towny#3109]
This ticket template is for bug/error reports. If you are suggesting a feature and not reporting a bug please delete all the text here and begin your ticket Title with Suggestion:

### What steps will reproduce the problem?
1. Typing /town Liminia
2.
3.

### What is the expected output?
1. Should get data on that town, a null value is returned instead. The town still shows up on this list of towns, and the claims are still intact.
2.
3.

### What is your Towny version?
Do not say "latest".
0.92.09

### What is your spigot/craftbukkit version?
Do not say "latest".
1.13.2

### Please use Pastebin.com to link the following files:

**Skipping these files is not an option. It is the easiest way to diagnose an issue ticket properly. If you do skip this step then your ticket will most likely be deleted and you will be asked to resubmit.**
1. Your full server startup from the logs\latest.log : https://pastebin.com/6J2GTCQ4
2. Your Towny config.yml (if using MySQL please remove password,) : https://pastebin.com/erV8NhPP
3. Your townyperms.yml : https://pastebin.com/kvdHgSrm
4. Your log's error : No error file in Towny's log folder.

(optional) If this is to do with permissions, 
5. Your permissions file :

(optional) If this is to do with chat,
6. Your chatconfig.yml :
7. Your channels.yml :

**Comment by LlmDl** [comment]
your essential plugin isnt loading , causing towny to not see the bank properly

**Comment by KingBohica** [comment]
Hmm, ok will look into it. It's strange that it is only affecting 1 town.

**Comment by KingBohica** [comment]
If it matters, the town in question was the only town so far to form a nation. The nation population dropped below the threshold to keep a nation alive and was automatically disbanded. This occurred around the same time the the town, which was the capital, started having issues.

**Comment by KingBohica** [comment]
The Nation in question still shows up on the nation list, but also generates null values and exceptions when trying to get data on it:
https://pastebin.com/XjengBpD

**Comment by KingBohica** [comment]
No worries, thanks for reexamining the post. If you mean the logs from the log folder in the root directory, there's not much to see there. Towny scans for old residents, deletes residents, and says resident purge complete. No error or mentions of nations disbanding that I can see, unless I'm on the wrong days. Are there other log folders you had in mind?

I don't believe he had gotten to the point of getting other towns to join the nation. It was just the one town. We have the minimum players set to 5 to form a nation, and his town dropped to 4 which then auto disbands the nation, I had thought.

You mentioned earlier when you thought Essentials was the culprit that it couldn't see the bank. The only thing I can think of dealing with nations and economy is we did change a default setting:
 refund_disband_low_residents: 'false'
Originally I guess they would get their money back that they had invested in forming a nation, should it auto-disband. We decided to disable this.

I know I'm not giving you much, but all I have is anecdotal evidence. If there's anything more I can do on my end, don't hesitate to ask. =p
